inlanecruising_plugin/src/inlanecruising_plugin.cpp

```diff
@@ -313,17 +256,14 @@ std::vector<cav_msgs::TrajectoryPlanPoint> InLaneCruisingPlugin::compose_traject
 
   log::printDoublesPerLineWithPrefix("yaw_values[i]: ", yaw_values);
 
-  for (int i = 0; i < yaw_values.size() - 1; i++)
-  {  // Drop last point
-    final_yaw_values.push_back(yaw_values[i]);
-    all_sampling_points.push_back(sampling_points[i]);
-  }
-
+  // Drop last point
```

Comment:
Is it still necessary to drop the last point when subcurves are no longer a thing? I think this logic was originally meant to support the concatenation of the sub-curves.

Reply:
ah okay. I don't think it is necessary then